homeassistant/components/influxdb.py

```diff
@@ -128,15 +132,17 @@ def influx_event_listener(event):
             _state = state.state
             _state_key = "state"
 
-        if override_measurement:
-            measurement = override_measurement
-        else:
-            measurement = state.attributes.get('unit_of_measurement')
-            if measurement in (None, ''):
-                if default_measurement:
-                    measurement = default_measurement
-                else:
-                    measurement = state.entity_id
+        measurement = state.attributes.get(CONF_INFLUXDB_MEASUREMENT)
```

Comment:
Add a comment before this line, explaining that this attributes can be set by a user in `customize`.